2024-12-17T08:25:32.930431+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '11cf8a87-db73-42f2-aac1-213617528ea0', 'counter': 1, 'stdout': '\x1b[0;31mERROR! We were unable to read either as JSON nor YAML, these are the errors we got from each:\x1b[0m', 'start_line': 0, 'end_line': 1, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.923501+00:00'}
2024-12-17T08:25:32.930920+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'a83e092d-e90d-43f3-8e71-31d1e0cda35e', 'counter': 2, 'stdout': '\x1b[0;31mJSON: Expecting value: line 1 column 1 (char 0)\x1b[0m', 'start_line': 1, 'end_line': 2, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.923738+00:00'}
2024-12-17T08:25:32.930990+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'a5cc0ce9-a0eb-4940-9242-9d464789591a', 'counter': 3, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 2, 'end_line': 3, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.923799+00:00'}
2024-12-17T08:25:32.931045+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'dd3f8b2b-fd26-4148-91c0-98ed597b9732', 'counter': 4, 'stdout': '\x1b[0;31mSyntax Error while loading YAML.\x1b[0m', 'start_line': 3, 'end_line': 4, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.923851+00:00'}
2024-12-17T08:25:32.931101+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'a04d6dd8-c1ea-4b3b-b5d5-3477f6b6c338', 'counter': 5, 'stdout': "\x1b[0;31m  did not find expected '-' indicator\x1b[0m", 'start_line': 4, 'end_line': 5, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.923898+00:00'}
2024-12-17T08:25:32.931151+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '517c78d1-9675-42da-898e-bd90869943bd', 'counter': 6, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 5, 'end_line': 6, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.923944+00:00'}
2024-12-17T08:25:32.931197+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '0bdb9596-5245-4876-b97b-5eafed0a9f48', 'counter': 7, 'stdout': "\x1b[0;31mThe error appears to be in '/home/thanhtut/test/ansible/git_tutorial/Chapter-6/playbook.yml': line 18, column 1, but may\x1b[0m", 'start_line': 6, 'end_line': 7, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.923987+00:00'}
2024-12-17T08:25:32.931251+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '43ddef13-e937-4701-aeae-783a59c07f0e', 'counter': 8, 'stdout': '\x1b[0;31mbe elsewhere in the file depending on the exact syntax problem.\x1b[0m', 'start_line': 7, 'end_line': 8, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.924040+00:00'}
2024-12-17T08:25:32.931300+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '4922a6e0-ea52-4747-bffd-053e8428963f', 'counter': 9, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 8, 'end_line': 9, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.924083+00:00'}
2024-12-17T08:25:32.931344+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '56ae4fbc-dd1a-4019-9998-5598407ee62c', 'counter': 10, 'stdout': '\x1b[0;31mThe offending line appears to be:\x1b[0m', 'start_line': 9, 'end_line': 10, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.924126+00:00'}
2024-12-17T08:25:32.931399+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '396321a5-61d1-487d-b5cf-79ebf0650e51', 'counter': 11, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 10, 'end_line': 11, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.924166+00:00'}
2024-12-17T08:25:32.931444+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'ed60ca65-2e56-4f42-bcf7-094869cdd0f4', 'counter': 12, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 11, 'end_line': 12, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.924208+00:00'}
2024-12-17T08:25:32.931485+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '9ca769ac-c0ba-40df-ad09-5084a7b25689', 'counter': 13, 'stdout': '\x1b[0;31mpre_tasks:\x1b[0m', 'start_line': 12, 'end_line': 13, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.924250+00:00'}
2024-12-17T08:25:32.931536+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'd8c6713c-e2ae-4f70-b6c0-ee9da7fff362', 'counter': 14, 'stdout': '\x1b[0;31m^ here\x1b[0m', 'start_line': 13, 'end_line': 14, 'runner_ident': '0ed2e5bf-2e8c-4532-8f83-8441d7e8db2f', 'created': '2024-12-17T08:25:32.924295+00:00'}
2024-12-17T12:57:56.899871+00:00 WARNING 'ansible_navigator.run' Dirty exit, killing the pid
2024-12-17T14:15:54.591662+00:00 ERROR 'ansible_navigator.actions.doc._run_runner' Error occurred while fetching doc for plugin stdout: '[WARNING]: stdout was not found
'
